src/main/java/com/spotify/docker/client/messages/Device.java

```diff
@@ -28,24 +28,49 @@
 import com.fasterxml.jackson.annotation.JsonProperty;
 import com.google.auto.value.AutoValue;
 
+import javax.annotation.Nullable;
+
 @AutoValue
 @JsonAutoDetect(fieldVisibility = ANY, getterVisibility = NONE, setterVisibility = NONE)
 public abstract class Device {
 
+  @Nullable
```

Comment:
dockerd didn't complain when I didn't set this or `pathInContainer`. I'm not sure if it makes sense for these to be nullable though. Should we enforce in code?